samples/net/sockets/echo_server/overlay-qemu_cortex_a53-e1000.conf

```diff
@@ -0,0 +1,18 @@
+CONFIG_ETH_E1000=y
```

Comment:
Regarding this file - wouldn't it make sense to simply have `boards/qemu_cortex_a53.conf`, so that the config is applied automatically? As far as I understood, there's no other option for networking on this platfrom than e1000.

Reply:
Good point, this is a sensible default option for networking on this platform.
I will look into it